### ISSUE TITLE
[Feature Request] Persistent browser data & ability to login to browser (google chrome)

### ISSUE THREAD

**Issue by DJKenzoDE** [m1k1o/neko#76]
good day everyone

how it is possible to login in google account on the google chrome browser?

best wishes from berlin

**Comment by m1k1o** [comment]
Hi, yes it is possible. Although, in order to keep it persistent between container recreates, browser data must be mounted as a volume.

**Comment by DJKenzoDE** [comment]
how to make is persistent? and is this solution good for the neko-rooms that mounted in the new room window?

**Comment by m1k1o** [comment]
You can mount volume to this folder inside container `/home/neko/.config/google-chrome`. Although, as i mentioned, preferences would, there would need to be adjusted. I am not sure right now, what everything needs to be changed (allowing cookies, allowing gistory etc.) but you can, again, mount custom policy file to `/etc/opt/chrome/policies/managed/policies.json`.

Yes, also neko-rooms supports volumes. **Private mount** would be best fit for your browser data and **template mount** would be good for preferences file. I have this on my roadmap in neko-rooms to allow having persistent rooms. First step - mounts - is already done. Just some frontendwork is missing, to make the UX better.

**Comment by DJKenzoDE** [comment]
how to make it possible on neko-rooms? only link the volume to your path?(etc/opt/...) and then is all done?

**Comment by m1k1o** [comment]
Yes, you only need to mount volume that you want to keep persistent. Explained in your other issue here: https://github.com/m1k1o/neko-rooms/issues/22

**Comment by bladecell** [comment]
I am trying to make logins persistent after container restart and I have /home/neko/.config/vivaldi/ and policies.json mounted on host and in policies.json I have these lines
`  "DefaultCookiesSetting": 1,
  "BrowserSignin": 1, 
  "RestoreOnStartup": 1`
but on every restart I lose logins.

**Comment by m1k1o** [comment]
Can you check if path is mounted correctly for policies at `/etc/opt/vivaldi/policies/managed/policies.json`, that it contains valid JSON (so no mistake has been made, like leaving `,` at the end of array). Also can you check in settings if thats allowed, because [default preferences as well deny it](https://github.com/m1k1o/neko/blob/50e5483661160e33267b5a4921648846736eb681/.docker/vivaldi/preferences.json#L221).

**Comment by m1k1o** [comment]
Added to docs.
